style.css

```diff
@@ -239,3 +237,55 @@ nav i{
     top:26rem;
 }
 
+.auth-container{
+    height:80vh;
+    width:100%;
```

Comment:
I would suggest to give proper spacing here (between height and 80vh) and could be consistent with it throughout the code.

Reply:
okay